samples/benchmark/bunnymark/Test.tscn

```diff
@@ -0,0 +1,10 @@
+[gd_scene load_steps=2 format=2]
+
+[ext_resource path="res://benchmarks/BunnymarkV1DrawTexture/d/BunnymarkV1DrawTexture.gdns" type="Script" id=1]
+
+[node name="Test" type="Node2D"]
+
+script = ExtResource( 1 )
+_sections_unfolded = [ "Material", "Visibility", "Z" ]
+
+
```

Comment:
Is this one needed?

Reply:
I guess it is, as it seems to be the benchmark scene ?

Reply:
true i didn't pay enough attention